test/test_helpers/pact_helper.rb

```diff
@@ -1,13 +1,26 @@
 PUBLISHING_API_PORT = 3001
+ORGANISATION_API_PORT = 4000
```

Comment:
Is this just arbitrary?

Reply:
I couldn't find any documentation on it, so yes I just picked 4000.

Reply:
Yep will do.

Reply:
https://github.com/alphagov/collections/pull/2278

Reply:
:please: :thankyou: